lua/telescope/builtin.lua

```diff
@@ -833,4 +833,34 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.tagstack = function(opts)
+  opts = opts or {}
+  local tagstack = vim.fn.gettagstack()
+  if vim.tbl_isempty(tagstack.items) then
+    return
+  end
+
+  for i, value in pairs(tagstack.items) do
```

Comment:
you should use `ipairs` when iterating over an array-like table.

Reply:
af3c348ce302fa71f9287aa3a2ed3668290c7274